test/fixtures.js

```diff
@@ -64,6 +64,18 @@ module.exports = {
     log.toString(),
     'toString'
   ],
+  'arrow function': [
+    (a) => a + 1,
+    '(a) => a + 1'
+  ],
+  'shorthand method': [
+    { key (a) { return a + 1 } },
+    '{key: function key (a) { return a + 1 }}'
+  ],
+  'arrow function in object': [
+    { key: (a) => a + 1 },
+    '{key: (a) => a + 1}'
+  ],
```

Comment:
I think we need add to test `{ method(a) { return a + 1; }}`

Reply:
Please check the test above in L72.